### PR TITLE
feat: add extension menu

### PR DESCRIPTION
**Pull request by fmoessbauer**
The  click-to-enable / disable logic of the extension icon did not follow extension development guidelines, as it was not clear what
happens when clicking the icon.

To fix this, we switch to a menu-based approach. In the future, this menu can be extended to also switch between accounts / identities.

Signed-off-by: Felix Moessbauer <felix.moessbauer@siemens.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,3 +108,5 @@ In case the extension is not working, check the following:
 
 This project is licensed according to the terms of the Mozilla Public
 License, v. 2.0. A copy of the license is provided in `LICENSES/MPL-2.0.txt`.
+We further use icons from the Material Design Icons, which are licensed under the Apache 2.0 License.
+A copy of the license is provided in `LICENSES/Apache-2.0.txt`.
```
Comment (jan-kiszka) [inline README.md:112]: Adding yet another license just for some simple icon - is that really worth it? Have you studied what the result of the combination could be license-wise?

Comment (fmoessbauer) [reply README.md:112]: These licenses can be combined without any problems. Anyways, I also thought about if that is really necessary. Let's see if we can find a MIT licensed icon library and just use that instead.

Comment (fmoessbauer) [reply README.md]: I replaced the icon with one from the yandex-ui-icons collection, which is MPL-2.0 licensed.

**Review by jan-kiszka — APPROVED** [review]
Looks good, works well

**Review by jan-kiszka — CHANGES_REQUESTED** [review]
Wait! I think the state machine of the menu can get out of sync, suggesting that you are guest while you are actually logged in. I think you can reproduce that by clicking on your account in the menu while you are logged in - although that should be a nop.

**Comment by fmoessbauer** [comment]
> Wait! I think the state machine of the menu can get out of sync, suggesting that you are guest while you are actually logged in. I think you can reproduce that by clicking on your account in the menu while you are logged in - although that should be a nop.

Good catch! While fixing this, I also added a spinner that is shown during in-flight operations. Sometimes it takes some seconds to login again. Technically the re-login is not required anyways, but this will be fixed in a separate MR.

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**